base-containers/base/bin/_run_student_intern

```diff
@@ -64,6 +62,7 @@ while start_cmd is None:
     for obj in unpacker:
         start_cmd = obj
 print("Unpacked start cmd")
+user = start_cmd["user"]  # Know if you are worker (on docker) or root (on Kata)
```

Comment:
Maybe add a check here that we are indeed allowed to start "root" in this container.
Is it possible for a compromised grading container to start a root student container on Docker?

Reply:
(it is indeed currently possible since this message comes from the grading container)

Reply:
Just after getting the user from the grading_container message, adding something like:
`if user=="root" and os.path.exists("/.__input/__shared_kernel"): exit`
would do the trick?

This way even in the case of a compromised grading_container manually sending a message asking to run as root, it would not if student container is using docker... no ?